src/v3/src/components/AppStoreDownloadLink/AppStoreDownloadLink.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box } from '@mui/material';
+import { Link } from '@okta/odyssey-react-mui';
+import { AppStoreDownloadLinkElement, UISchemaElementComponent } from 'src/types';
+
+import Logger from '../../../../util/Logger';
+
+const AppStoreDownloadLink: UISchemaElementComponent<{
+  uischema: AppStoreDownloadLinkElement
+}> = ({ uischema }) => {
+  const { options } = uischema;
+  const platform = options.platform ? options.platform.toLowerCase() : '';
+  let imageSrc = '/img/appstore/';
+  switch (platform.toLowerCase()) {
+    case 'android':
+      imageSrc += 'google-play-store.svg';
+      break;
+
+    case 'ios':
+      imageSrc += 'apple-app-store.svg';
+      break;
+
+    case 'windows':
+      imageSrc += 'windows-app-store.svg';
+      break;
+
+    case 'osx':
+      imageSrc += 'osx-app-store.svg';
+      break;
+
+    default:
+      Logger.error('Unsupported platform: ', platform);
+  }
+
+  return (
+    <Box>
+      <Link
+        href={options.url}
+        rel="noopener noreferrer"
+        ariaLabel={options.altText}
+        className="app-store-link"
+      >
+        <Box
```

Comment:
Can you copy over the the code I linked from our odyssey-1.x branch for the [new Image functional component](https://github.com/okta/okta-signin-widget/blob/odyssey-1.x/src/v3/src/components/Image/Image.tsx) that we created there and then use it in place of this `<Box component="img"... />`? 

That component already has all of the props you use here and it would save us some time from having to convert it over to use `Image` ourselves :pray:

Reply:
@trevoring-okta  I've created a follow up jira OKTA-701606 for this change